packages/headless/src/features/generated-answer/generated-answer-actions.ts

```diff
@@ -19,6 +20,7 @@ import {
   SearchSection,
 } from '../../state/state-sections';
 import {validatePayload} from '../../utils/validate-payload';
+import {logGeneratedAnswerStreamEnd} from './generated-answer-analytics-actions';
 import {buildStreamingRequest} from './generated-awswer-request';
```

Comment:
unrelated to this PR but this file has a mistake in its name.
```suggestion
import {buildStreamingRequest} from './generated-answer-request';
```

Reply:
Also unrelated, in this file, there is a useless import at line 3.

Reply:
BooleanValue? it is used. 😅 which import are you referring to?

Reply:
Oops sorry ! I meant line 3 in `generated-answer-slice.ts`.